src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
The class name `stars--0` should follow the BEM convention. Consider using a more descriptive modifier name, such as `stars--level-0`.